src/Servers/HttpSys/src/Microsoft.AspNetCore.Server.HttpSys.csproj

```diff
@@ -14,6 +14,15 @@
     <NoWarn>$(NoWarn);CA1416</NoWarn>
   </PropertyGroup>
 
+
+  <ItemGroup>
+    <None Include="$(SharedSourceRoot)HttpSys\RequestProcessing\RequestHeaders.Generated.tt" Link="RequestProcessing\RequestHeaders.Generated.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+      <LastGenOutput>RequestHeaders.Generated.cs</LastGenOutput>
+    </None>
+    <Service Include="{508349b6-6b84-4df5-91f0-309beebad82d}" />
```

Comment:
This is new to me - what does it do?  Maybe it's new in Core and I've only used tt in Framework projects?

Reply:
Huh, I guess T4 became opt-in at some point and I missed it.